editor/modes/visual-editor/index.js

```diff
@@ -2,12 +2,14 @@
  * External dependencies
  */
 import { connect } from 'react-redux';
+import { first, last } from 'lodash';
 
 /**
  * WordPress dependencies
  */
 import { __ } from 'i18n';
 import { Component, findDOMNode } from 'element';
+import { LETTER_A, SMALL_LETTER_A } from 'utils/keycodes';
```

Comment:
Actually, I'm going to use only LETTER_A because there's no difference between the two in keydown events. (The keycode is the same)